src/Framework/FieldsAPI/FormField/FieldOption.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\FormField;
+
+/**
+ * Class FieldOption
+ *
+ * @unreleased
+ */
+class FieldOption {
```

Comment:
Implement the [JsonSerializable](https://www.php.net/manual/en/class.jsonserializable.php) interface and tweak the method name (camelcase), then it will be official. 😎

Reply:
Can do! The `FormField` class is missing those as well.

Reply:
Oh, I see that’s because `Node` implements it.

Reply:
Resolved in 192196c